helios-services/src/test/java/com/spotify/helios/agent/TaskRunnerTest.java

```diff
@@ -58,6 +69,11 @@
   @Mock private Clock clock;
   @Mock private ContainerDecorator containerDecorator;
 
+  @Mock private TaskRunner.NopListener mockListener;
```

Comment:
@negz: Yeah you can't use the `@Mock` annotation for that. It's simple to do nonetheless.

Just do `final MyClass myMock = Mockito.mock(MyClass.class);` (bonus points for doing a static import of that function to make it a bit more terse`.

This is basically what "@Mock" does under the hoods.